CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+v1.2.1
+
+* Change to tracking replacement so it accepts strings or regular expression as a matching condition.

```

Comment:
Could we please:

1) write these from a user's perspective rather than implementation detail
2) make it clear they are bug fixes (since this is a patch release)
3) refer to any fixed issues

e.g. (tweak to your linking)
Fix Jekyll front matter being spellchecked if it contained a regex (#107)
Fix content between Jekyll front matter and in-content headings not being spellchecked